services/watson-extension/src/watson_extension/core/insights/notifications.py

```diff
@@ -0,0 +1,48 @@
+import injector
+import uuid
+from datetime import datetime
+
+import watson_extension.config as config
+from watson_extension.clients.insights.notifications import NotificationsClient
+
+
+class NotificationsCore:
+    def __init__(self, notifications_client: injector.Inject[NotificationsClient]):
+        self.notifications_client = notifications_client
+
+    async def send_rbac_request_admin(
+        self,
+        org_id: str,
+        username: str,
+        user_email: str,
+        user_message: str,
+        requested_url: str,
+    ):
+        if config.is_running_locally:
```

Comment:
Lets try to remove global values from here. 

Instead of this I suggest to create a  `NotificationsClientNoOp` that just does nothing and connect it on `startup.py` when we are running locally (like we do with the `FixedUserIdentityProvider`)

Reply:
I think that going forward we could benefit from not having `config` as a global, it depends on envs and is just harder to interact with it on tests.

Reply:
Thanks for the suggestion, tests are now passing!